sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py

```diff
@@ -465,6 +465,11 @@ async def download_blob(self, offset=None, length=None, **kwargs):
             The number of parallel connections with which to download.
         :keyword str encoding:
             Encoding to decode the downloaded bytes. Default is None, i.e. no decoding.
+        :keyword progress_hook:
```

Comment:
Is this an optional kwarg? If so, just curious as to why this convention as opposed to:
`:keyword Callable[[int, int], Awaitable[None]] progress_hook`
Same feedback on non-async client

Reply:
The API View generator did not like this type listed in that format for some reason. It would cause this keyword to not show up at all in the View. So, I moved it to a `paramtype` to appease API View. Good question though as I would normally like to have it how you mentioned.